ambari-server/src/main/java/org/apache/ambari/server/actionmanager/Request.java

```diff
@@ -96,6 +98,7 @@ public Request(@Assisted long requestId, @Assisted("clusterId") Long clusterId,
     this.endTime = -1;
     this.exclusive = false;
     this.clusterHostInfo = "{}";
+    this.userName = AuthorizationHelper.getAuthenticatedName();
```

Comment:
Since this is a part of the constructor, care should be taken to allow for nulls for a background operation lets say that is not initiated by a user.

Reply:
AuthorizationHelper.getAuthenticatedName() returned "internal" in such cases, as far as I could see. 
Should I just allow nulls?
Should I move the AuthorizationHelper.getAuthenticatedName() call out of the constructor?

Reply:
Oh, you were quick to get to this while I was still reviewing :-)
I think nulls are ok, maybe other reviewers can chime in.